### ISSUE TITLE
Mypy flags comment as invalid

### ISSUE THREAD

**Issue by DavidFreire-FEUP** [sffjunkie/astral#79]
Hi! Not here to nag you or ask for features, just wanted to ask if this error also occurs to the developers here when you run mypy against atral's code:
```/astral/sun.py:450: error: Invalid "type: ignore" comment```

Running mypy="0.971" and astral="^3.1". Thanks!
https://github.com/sffjunkie/astral/blob/b805285654948c6513f378952aab685085efbad6/src/astral/sun.py#L447

**Comment by sondrelg** [comment]
Yes, upgrading to the latest version of astral broke our CI pipeline with

````
.venv/lib/python3.10/site-packages/astral/sun.py:447: error: Invalid "type: ignore" comment  [syntax]
